app/client/package.json

```diff
@@ -9,7 +9,12 @@
         "build": "next build",
         "relay": "relay-compiler --watch",
         "typecheck": "tsc -p tsconfig.prod.json --noEmit",
-        "lint": "eslint \"src/**\""
+        "lint": "eslint \"src/**\"",
+        "test:e2e": "playwright test && yarn allure:generate-report",
+        "test:e2e:ci": "dotenv -v CI=1 -- playwright test && yarn allure:generate-report",
+        "test:e2e:headed": "playwright test --headed && yarn allure:generate-report",
+        "allure:generate-report": "allure generate ./allure-results --clean",
+        "allure:report": "allure open ./allure-report"
```

Comment:
I think it may be better to have a `ci:test` command, which will run all of the appropriate e2e tests and in the future also run the unit tests.

Reply:
We can just add that in when we do have those tests and have it call test:e2e as part of it though. All we'd have to do is update the command called in the workflow.